spec/ParseQuery.spec.js

```diff
@@ -3157,23 +3157,28 @@ describe('Parse.Query testing', () => {
         ok(!result.dirty(), 'expected result not to be dirty');
         strictEqual(result.get('foo'), 'baz');
         strictEqual(result.get('bar'), undefined, "expected 'bar' field to be unset");
+        strictEqual(result.get('qux'), undefined, "expected 'qux' field to be unset");
         return result.fetch();
       })
       .then(function (result) {
         strictEqual(result.get('foo'), 'baz');
         strictEqual(result.get('bar'), 1);
+        strictEqual(result.get('qux'), 2);
       })
       .then(function () {
         obj._clearServerData();
         const query = new Parse.Query(TestObject);
-        query.select([]);
```

Comment:
This is the test case I **removed** because it 1) doesn't work anymore 2) not sure why we want this particular functionality for `select`. If you really want to exclude all keys, there should probably be an `excludeAll` or an `exclude(["*"]`

Reply:
I think we have to be careful in removing a test case. I actually just fixed a Parse Server issue for `select([])` some weeks ago and I would like to better understand what the issue around this PR does and why you say this doesn't work anymore. I would like to have this discussion in a separate Parse Server issue.